Sources/Kitura/RouterResponse.swift

```diff
@@ -128,10 +130,14 @@ public class RouterResponse {
     ///                    working with.
     /// - Parameter request: The `RouterRequest` object that is paired with this
     ///                     `RouterResponse` object.
-    init(response: ServerResponse, routerStack: Stack<Router>, request: RouterRequest) {
+    /// - Parameter encoder: The `BodyEncoder` that will be used the encode the request body.
+    /// - Parameter mediaType: The `MediaType` the media type which will be sent in the response "Content-Type" header.
+    init(response: ServerResponse, routerStack: Stack<Router>, request: RouterRequest, encoder: BodyEncoder = JSONEncoder(), mediaType: MediaType = .json) {
```

Comment:
Could `encoder` and `mediaType` be a tuple? (you could pass straight through the one you got from `selectResponseEncoder()`)
It would make it clearer that the two are closely related.